src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/TaskController.java

```diff
@@ -224,6 +227,7 @@ public TaskController(CamundaService camundaService,
             message = INTERNAL_SERVER_ERROR
         )
     })
+    @ResponseStatus(HttpStatus.NO_CONTENT) //Fixes incorrect Swagger 200 response code
     @PostMapping(path = "/{task-id}/unclaim",
         produces = APPLICATION_JSON_VALUE)
     public ResponseEntity<String> unclaimTask(@RequestHeader("Authorization") String authToken,
```

Comment:
Added as No Content, but returning String

Reply:
true should not return anything